lib/server/Server.js

```diff
@@ -19,6 +20,7 @@ var Server = {
         var onRouteHandled = config.onRouteHandled;
         var ServerAppToUse = config.ServerApp || ServerApp;
         var appRoot = environmentConfig.appRoot;
+        var proxyClientRequestsToApi = config.proxyClientRequestsToApi;
```

Comment:
My argument is just on the basis of confusion with the server sending requests to API_HOST and the client sending them to the app, not the API_HOST. Both should end up in the same location by default in my opinion.